editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/restconfiguration/RestEditorTestIT.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.fusesource.ide.camel.editor.restconfiguration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.eclipse.jface.viewers.StructuredSelection;
+import org.eclipse.ui.IEditorPart;
+import org.fusesource.ide.camel.editor.CamelEditor;
+import org.fusesource.ide.camel.model.service.core.model.RestElement;
+import org.fusesource.ide.camel.model.service.core.model.RestVerbElement;
+import org.fusesource.ide.camel.test.util.editor.AbstractCamelEditorIT;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * @author brianf
+ *
+ * Load a Camel config with some Rest DSL
+ * See if the Rest tab is there
+ * Switch to the Rest tab <-- stopped here, but the rest is next
+ * See if the first rest element is there (getSelection should by default return it)
+ * Select a second rest element
+ * Switch to Source tab
+ * Switch back to Rest tab
+ * See if the second rest element is still selected
+ * repeat with REST operations
+ * 
+ */
+@RunWith(Parameterized.class)
+public class RestEditorTestIT extends AbstractCamelEditorIT {
+
+	@Parameters(name = "Route container of type: {0}")
+	public static String[] data() {
+	    return new String[] { "camelContext" };
```

Comment:
there is a single parameter, is it because you plan to add other parameters soon? Otherwise a non-parameterized would be simpler.